### PR TITLE
adding in StarRocks connection information

### PR DESCRIPTION
**Pull request by alberttwong**
## What is the purpose of the pull request

adding in StarRocks connection information

## Brief change log

documentation to the website

## Verify this pull request

This pull request is a trivial rework / code cleanup without any test coverage.

### CODE REVIEW

**Comment by sagarlakshmipathy** [comment]
Thanks for submitting the PR @alberttwong, I'll review this over the weekend and comment/approve.

File: website/docs/starrocks.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+{ label: 'targetFormat: HUDI', value: 'hudi', },
+{ label: 'targetFormat: DELTA', value: 'delta', },
+{ label: 'targetFormat: ICEBERG', value: 'iceberg', },
+]}
+>
+<TabItem value="hudi">
+
+:::tip Note:
+If you are following the example from [Hive Metastore](/docs/hms), you can query the OneTable synced Hudi table
+from Trino using the below query.
```
Comment (sagarlakshmipathy) [inline website/docs/starrocks.md:40]: Refers to the Trino example, likely copy-paste miss.

File: website/docs/starrocks.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+SELECT * FROM hudi.hudi_db.<table_name>;
+```
+:::
+
+</TabItem>
+<TabItem value="delta">
+
+:::tip Note:
+If you are following the example from [Hive Metastore](/docs/hms), you can query the OneTable synced Delta table
+from Trino using the below query.
```
Comment (sagarlakshmipathy) [inline website/docs/starrocks.md:51]: Same as Hudi tab above.

File: website/docs/starrocks.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+SELECT * FROM delta.delta_db.<table_name>;
+```
+:::
+
+</TabItem>
+<TabItem value="iceberg">
+
+:::tip Note:
+If you are following the example from [Hive Metastore](/docs/hms), you can query the OneTable synced Iceberg table
+from Trino using the below query.
```
Comment (sagarlakshmipathy) [inline website/docs/starrocks.md:62]: Same as Hudi tab above.

File: website/docs/starrocks.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+* [Hudi Catalog](https://docs.starrocks.io/docs/data_source/catalog/hudi_catalog/)
+* [Delta Lake Catalog](https://docs.starrocks.io/docs/data_source/catalog/deltalake_catalog/)
+* [Iceberg Catalog](https://docs.starrocks.io/docs/data_source/catalog/iceberg_catalog/)
+
+For hands on experimentation, please follow [Creating your first interoperable table](/docs/how-to#create-dataset)
+to create OneTable synced tables followed by [Hive Metastore](/docs/hms) to register the target table
+in Hive Metastore. Once done, please follow the below high level steps:
+1. Start the StarRocks server 
+2. From the directory where you have installed mysql-cli: login by running `mysql -P 9030 -h 127.0.0.1 -u root --prompt="StarRocks > "`.
+3. Add the external catalog (slight variation between open table formats)  `CREATE EXTERNAL CATALOG hudi_catalog_hms PROPERTIES ("type" = "hudi","hive.metastore.uris" = "thrift://hivemetastore:9083" );`.
```
Comment (sagarlakshmipathy) [inline website/docs/starrocks.md:24]: Can you put points 3 and 4 inside the `Tabs` block below aligning the examples for all three formats?

File: website/docs/starrocks.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
+---
+sidebar_position: 8
```
Comment (sagarlakshmipathy) [inline website/docs/starrocks.md:2]: You also need to modify the sidebars.js file to actually see the option to select the StarRocks query engine on the site. Refer to the readme [here](https://github.com/onetable-io/onetable/tree/main/website#docs).

**Review by sagarlakshmipathy — COMMENTED** [review]
Left some comments. Thanks for submitting the PR again.

**Comment by alberttwong** [comment]
fixed and squashed.

**Review by sagarlakshmipathy — APPROVED** [review]
LGTM

**Pull request merged**